config/envs/unit_test.py

```diff
@@ -57,3 +59,6 @@ class UnitTestConfig(DefaultConfig):
     # Which is overkill for now. 28/06/2024.
     CELERY = DefaultConfig.CELERY
     CELERY["task_always_eager"] = True
+
+    E2E_NOTIFY_FIND_API_KEY = os.environ.get("E2E_NOTIFY_FIND_API_KEY")
+    E2E_DEVTEST_BASIC_AUTH = os.environ.get("E2E_DEVTEST_BASIC_AUTH")
```

Comment:
I got a failure:
```
E           playwright._impl._errors.Error: Page.goto: net::ERR_INVALID_AUTH_CREDENTIALS at https://None@find-monitoring-data.dev.access-funding.test.levellingup.gov.uk/download
E           Call log:
E           navigating to "https://None@find-monitoring-data.dev.access-funding.test.levellingup.gov.uk/download", waiting until "load"
```
Should we do a hard look up if `e2e` is passed as a param to check this is set? Also, should we separate out the username and passsword, so people don't need to know the syntax?

Reply:
This is in config so we can't do hardlookups or check on pytest params.

I'll add an example as a default so that it's a bit more obvious what format